src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -253,7 +254,7 @@ protected override void OnScriptFileChanged(object sender, FileSystemEventArgs e
             }
         }
 
-        private async Task<Dictionary<string, object>> CreateScriptExecutionContextAsync(object input, DataType dataType, TraceWriter traceWriter, FunctionInvocationContext invocationContext)
+        private async Task<Dictionary<string, object>> CreateScriptExecutionContextAsync(object input, DataType dataType, TraceWriter traceWriter, ILogger logger, FunctionInvocationContext invocationContext)
```

Comment:
Why not just pull the ILogger from the FunctionInvocationContext, instead of adding a new parameter?